src/swarm-scenario/Swarm/Game/World/Render.hs

```diff
@@ -38,8 +40,34 @@ import Swarm.Language.Pretty (prettyString)
 import Swarm.Util (surfaceEmpty)
 import Swarm.Util.Content
 import Swarm.Util.Erasable (erasableToMaybe)
+import Swarm.Util.Yaml
 import System.IO (hPutStrLn, stderr)
 
+newtype OneBitColor = OneBitColor Bool
```

Comment:
@kostmo could you please add an explicit export list, so that it's easier to see if any definitions are internal?